addons/web_editor/views/wysiwyg_iframe_content.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <template id="web_editor.wysiwyg_iframe_content">
+        <html>
+            <head>
+                <meta charset="utf-8" />
+                <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+                <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
+                <t t-foreach="jsLibs" t-as="item">
+                    <script type="text/javascript" t-att-src="item['src']"></script>
+                </t>
+                <t t-foreach="cssLibs" t-as="item">
+                    <link type="text/css" rel="stylesheet" t-att-href="item['href']"></link>
+                </t>
+                <t t-if="not readonly">
```

Comment:
why the t-if?

Reply:
As we discuss :) the xml template can be call with readonly option or not